IO/BeagleBone/PWMDeviceBBB.cs

```diff
@@ -38,65 +38,66 @@ public static class PWMBBB
         public static PWMDeviceBBB PWMDevice1 { get; private set; }
         public static PWMDeviceBBB PWMDevice2 { get; private set; }
 
-        /// <summary> Prepares the given PWM ports for use. Should only be called from BeagleBone.Initialize(). </summary>
-        static internal void Initialize(bool[] EnableBuses)
-        {
-            if (EnableBuses == null || EnableBuses.Length != 3) { throw new Exception("Invalid enable array given to PWMBBB.Initialize."); }
-            //                                                             A1            A2                             B1            B2
-            if (EnableBuses[0]) { PWMDevice0 = new PWMDeviceBBB(new BBBPin[] { BBBPin.P9_22, BBBPin.P9_31 }, new BBBPin[] { BBBPin.P9_21, BBBPin.P9_29 }); }
-            if (EnableBuses[1]) { PWMDevice1 = new PWMDeviceBBB(new BBBPin[] { BBBPin.P9_14, BBBPin.P8_36 }, new BBBPin[] { BBBPin.P9_16, BBBPin.P8_34 }); }
-            if (EnableBuses[2]) { PWMDevice2 = new PWMDeviceBBB(new BBBPin[] { BBBPin.P8_19, BBBPin.P8_45 }, new BBBPin[] { BBBPin.P8_13, BBBPin.P8_46 }); }
-        }
-
-        /// <summary> Converts a pin number to the corresponding PWM device and output number. Needed as every output is connected to 2 physical pins. </summary>
-        static internal PWMPortEnum PinToPWMID(BBBPin Pin)
+        public static PWMOutputBBB GetFromPin(BBBPin Pin)
```

Comment:
The only thing I could think to add is a comment here mentioning that the method could return null if the pin isn't a PWM output. User might find that information useful :)